packages/core/src/app/address/isValidStaticAddress.tsx

```diff
@@ -0,0 +1,21 @@
+import { Address, FormField } from "@bigcommerce/checkout-sdk";
+import { isEmpty } from "lodash";
+
+import isValidAddress from "./isValidAddress";
+
+export default function isValidStaticAddress(
```

Comment:
If I see this correctly, this is the old logic when the experiment is off?

Reply:
That's right, its the old logic, which used to display the empty option in address selector. Once the experiment is rolled out we may get rid of this logic and will only need to check if address is empty, since we want to show the address in the selector even if it is invalid.